Dockerfile.template

```diff
@@ -22,8 +20,7 @@ RUN JOBS=MAX npm install --production --unsafe-perm && npm cache verify && rm -r
 # This will copy all files in our root to the working  directory in the container
 COPY . ./
 
-# Enable systemd init system in container
-ENV INITSYSTEM on
+ENV UDEV=1
```

Comment:
Why enabling UDEV?

Reply:
Apparently to maintain behaviour of previous base images (being single service and privileged)